backend/main.py

```
@@ -18,4 +19,10 @@
 Base.metadata.create_all(bind=engine)
 
 # ✅ Include routers
-app.include_router(auth.router, prefix="/auth", tags=["Authentication"])
\ No newline at end of file
+app.include_router(auth.router, prefix="/auth", tags=["Authentication"])
+app.include_router(connections.router, prefix="/connections", tags=["Connections"])
+
+# ✅ Root Route
+@app.get("/")
+def root():
```

Ruff (0.8.2)

Missing return type annotation for public function `root`

(ANN201)